server_tests/test/clientTests.js

```diff
@@ -787,89 +789,151 @@ describe('RUST API TEST SUITE', function () {
     if (!isOSC2) {
       return this.skip()
     }
+    var expectedImageCount
+    var expectedVideoCount
+    var totalEntryCount
 
     before(function () {
       // delete exisiting files on SD card, if any
+      return Utility.deleteAllImages() //should have a new utility deleteAll() or something
```

Comment:
While I don't necessarily disagree about using the count... I don't think you can count on the card not being wiped when running the tests. Should there not at least be one test to make sure the list response is correct for an empty SD card?